ArduCopter/mode.h

```diff
@@ -282,7 +289,7 @@ class ModeAcro : public Mode {
     float throttle_hover() const override;
 
 private:
-
+    AirMode saved_airmode;
```

Comment:
Hi there, I wonder why we need to save and restore the air mode?

Reply:
I guess it's because there's also a switch and we want to override the switch using the parameter?  I worry a little bit that it could end up with a funny situation if the user moves the airmode switch while in acro mode.  Maybe it should use a combination of the parameter and the RC switch or just use the last one that changed... or maybe if an rc switch has been setup then it should just use that?